File: src/addon365.Database.Entity/Report/InquiryReport.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace addon365.Database.Entity.Report
{
    public class InquiryReport
    {
        [Key]
        public Int64 Id { get; set; }
        public DateTime Date { get; set; }
        public string Name { get; set; }
        public int Count { get; set; }
    }
}

File: src/DotNet/Should be removed/addon365.Web.API/ClientApp/src/app/models/inquiry-report.ts
export class InquiryReport {
  id: number;
  name: string;
  date: Date;
  count: number;

  toArray() {
    let array = new Array<any>();
    array.push(this.name);
    if (!this.date)
      array.push(this.date);
    array.push(this.count);
    return array;
  }
  copyFrom(obj: InquiryReport) {
    this.name = obj.name;
    if (!this.date) {
      this.date = new Date(obj.date);
    }
    this.count = obj.count;
  }

 
}


